front/src/views/PatientSignUpView/PatientSignUpView.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+
+import style from './patient_sign_up_view.scss';
+import { Button, TextInput, PasswordInput } from '../../components';
+import Svg from 'svg-inline-react';
+import registrationIcon from '../../../assets/icons/user-tie.svg';
+
+export class PatientSignUpView extends Component {
+
+  renderInputWithLabel(inputType, label) {
+    return (
+      <div className={ style['simple-input-box'] }>
+        <h3 className={ style['input-label'] }>
+            { label }:
+        </h3>
+        <div className={ style['input-container'] }>
+          <div className="row">
+          { (() => {
```

Comment:
No, we have components: TextInput and Password Input. I would leave it as it is, but this part with lambda function and switch is a kind of scary :D. So please for example do something like this:
`let input = {
text:<TextInput className="col-xs-12" />,
password: <PasswordInput className="col-xs-12" /> 
}`

and then you can use it this way in JSX: 
`{input[inputType]}`